app/assets/stylesheets/companies/companies_index.scss

```diff
@@ -0,0 +1,424 @@
+        margin-right: calc(8px + (10 - 8) * ((100vw - 1164px) / (1822 - 1164)));
+        margin-bottom: calc(20px + (30 - 20) * ((100vw - 1164px) / (1822 - 1164)));
+        margin-left: calc(8px + (10 - 8) * ((100vw - 1164px) / (1822 - 1164)));
+        font-weight: 500;
+        font-size: 33px;
+        line-height: 41px;
+        text-align:center;
+
+        &:hover {
+          color: $base-color;
+        }
+      }
+
+      .company-info-rows {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        justify-content: flex-start;
+        margin-top: calc(10px + (15 - 10) * ((100vw - 900px) / (1822 - 900)));
+        margin-right: calc(20px + (50 - 20) * ((100vw - 900px) / (1822 - 900)));
+        margin-bottom: 30px;
+        margin-left: calc(20px + (50 - 20) * ((100vw - 900px) / (1822 - 900)));
+      }
+
+      .company-info {
+        display: flex;
+        flex-direction: column;
+        width: 35%;
+
+        &:nth-child(1) {
+          margin-bottom: calc(10px + (20 - 10) * ((100vw - 900px) / (1822 - 900)));
+        }
+
+        &:nth-child(2) {
+          margin-left: calc(60px + (100 - 60) * ((100vw - 900px) / (1822 - 900)));
+          margin-bottom: calc(10px + (20 - 10) * ((100vw - 900px) / (1822 - 900)));
+        }
+
+        &:nth-child(4) {
+          margin-left: calc(60px + (100 - 60) * ((100vw - 900px) / (1822 - 900)));
+        }
+
+        .info-number {
+          display: flex;
+          align-items: center;
+          font-style: normal;
+          font-weight: 700;
+          font-size: calc(60px + (74 - 60) * ((100vw - 320px) / (1920 - 320)));
+          line-height: 87px;
+
+          &:hover {
+            color: $base-color;
+          }
+        }
+
+        .info-text {
+          display: flex;
+          align-items: center;
+          font-size: calc(11px + (15 - 11) * ((100vw - 320px) / (1920 - 320)));
+          line-height: 19px;
+          white-space: nowrap;
+
+          &:hover {
+            color: $base-color;
+          }
+        }
+      }
+    }
+
+    .container-about {
+      display: flex;
+      flex-wrap: nowrap;
+      align-items: center;
+      justify-content: center;
+      position: relative;
+      width: 760px;
+      height: 460px;
+      border: 10px solid $color-company;
+      cursor: auto;
+
+      .text-about {
+        display: flex;
+        align-items: center;
+        font-size: 23px;
+        line-height: 28px;
+        text-align: justify;
+
+        @media screen and (max-width: 1450px) {
+          font-size: calc(14px + (23 - 14) * ((100vw - 320px) / (1920 - 320)));
+        }
+      }
+    }
+  }
+
+  @media screen and (max-width: 1163px) {
+
+    .wrapper-company {
+
+      &:nth-child(odd) {
+
+        .container-about {
+          border-radius: 40px;
+        }
+      }
+
+      &:nth-child(even) {
+
+        .container-about {
+          border-radius: 40px;
```

Comment:
the same style rules for even and odd children, should we declare it separately?

Reply:
Unfortunately, when I put these styles in the general selector, these rules did not work